.github/CODEOWNERS

```diff
@@ -0,0 +1,7 @@
+# CODEOWNERS file for mitlib-tf-workloads-ecr
+
+################################################################################
+# This repository is maintained by the MIT Libraries Infrastructure Team.
+* @mitlibraries/infraeng-terraform-reviewers
+
+# There are no individual code owners for this repository.
```

Comment:
Mostly good. But, per the GitHub best practices (https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-and-branch-protection), there should be a line for controlling the `CODEOWNERS` file itself that should list @cabutlermit as the owner.

Reply:
I've updated the codeowners file.